tests/models/test_models_small.py

```diff
@@ -7,16 +7,43 @@
 """
 import pytest
 
+SKIPPED_MODEL_REASON = {
+    "allenai/OLMo-1B": "Hf side requirements",
+    "google/gemma-1.1-2b-it": "No bitwise correctness for fp32",
+    "Qwen/Qwen-1_8B": "No bitwise correctness for fp32"
+}
+
 MODELS = [
-    "facebook/opt-125m",
+    # baichuan          -> tested in medium
+    "bigscience/bloom-560m",
+    # chatglm           -> tested in medium
+    # command-r         -> not tested
+    # dbrx              -> not tested
+    # decilm            -> tested in medium
+    "deepseek-ai/deepseek-coder-1.3b-instruct",
+    # falcon            -> tested in medium
+    "google/gemma-1.1-2b-it",
     "gpt2",
     "bigcode/tiny_starcoder_py",
+    # gpt-j             -> tested in medium
     "EleutherAI/pythia-70m",
-    "bigscience/bloom-560m",  # Testing alibi slopes.
+    # internlm2         -> tested in medium
+    # jais              -> not tested
+    "TinyLlama/TinyLlama-1.1B-Chat-v1.0",
+    "openbmb/MiniCPM-2B-128k",
+    # mixtral           -> not tested
+    # mixtral-quant     -> not tested
+    # mpt               -> tested in medium
+    "allenai/OLMo-1B",
+    "facebook/opt-125m",
+    # orion             -> tested in medium
     "microsoft/phi-2",
-    "stabilityai/stablelm-3b-4e1t",
-    # "allenai/OLMo-1B",  # Broken
+    "Qwen/Qwen-1_8B",
+    "Qwen/Qwen1.5-1.8B",
+    # qwen2 moe         -> not tested
+    "stabilityai/stablelm-2-1_6b-chat",
     "bigcode/starcoder2-3b",
+    # xverse            -> tested in medium
 ]
```

Comment:
Several of these left in are still tested in medium. Obviously this test is more strict as it is measuring float exact token match, but do we need to duplicate models like bloom, deepseek, starcoder?